main.go

```diff
@@ -46,8 +46,9 @@ import (
 )
 
 const (
-	defaultNamespaceEnvVar = "DEFAULT_NAMESPACE"
-	watchNamespacesEnvVar  = "WATCH_NAMESPACES"
+	systemNamespaceEnvVar     = "SYSTEM_NAMESPACE"
+	monitoringNamespaceEnvVar = "MONITORING_NAMESPACE"
+	dbNamespacesEnvVar        = "DB_NAMESPACES"
```

Comment:
Are these values mandatory? If yes, are we missing validation for them?

Reply:
Yes and no.
We have the validation [here](https://github.com/percona/everest-operator/blob/e5ad5afe3859f9bb3210058a89463bb24a15b790/main.go#L82-L97).
But I just found that the last one is missing an exit :see_no_evil: I'll fix it.

Reply:
This part validates if the environment variable exists. It doesn't check if it's not empty, does not have restricted characters, etc.

Reply:
Ohh I see your point. We should check if the variable it's empty but we don't need to check for other characters or anything like that because it will fail to create the controller if the namespace doesn't exist and it will return an  appropriate message like this
```
[manager] 2024-02-06T13:54:40Z	ERROR	setup	unable to create controller	{"controller": "DatabaseCluster", "error": "namespaces \"awrongone\" not found"}
```